src/main/java/teammates/logic/core/GoogleCloudLoggingService.java

```diff
@@ -94,4 +108,154 @@
         return errorLogs;
     }
 
+    @Override
+    public void createFeedbackSessionLog(String courseId, String email, String fsName, String fslType)
+            throws LogServiceException {
+        String payload = "Feedback session log: course ID=" + courseId + ", email=" + email
+                + ", feedback session name=" + fsName + ", log type=" + fslType;
+        LogEntry entry = LogEntry.newBuilder(StringPayload.of(payload))
+                .setLogName(FEEDBACK_SESSION_LOG_NAME)
+                .addLabel(FEEDBACK_SESSION_LOG_COURSE_ID_LABEL, courseId)
+                .addLabel(FEEDBACK_SESSION_LOG_EMAIL_LABEL, email)
```

Comment:
A bit unrelated but what is the expected behaviour when a student changes their email?

Reply:
This is a good point. We don't even have a record of what the previous email was. I think the UI need to show the email address in a way it is clear that the tracking is done via the email. We have to be careful because instructors can take action based on the audit log that can affect student marks.
Also consider how we track activities registered (i.e., 'joined' using a Google account) students. What about students who registered halfway? It is also possible for a student to be de-register after a while (e.g., sometimes, users email me to say they can't login due to some Google account issue and I de-register them so that they can access using the access links instead). Does any of the above affects the tracking of activities?

Reply:
I think this has more to do with how we go about parsing the logs and displaying information to the user. Important considerations before the feature is released to the users definitely. For now with this PR, we will just log whatever email was used to access the submission form if I'm not mistaken. We should be okay going ahead with deploying this and then figuring out how to parse/display later on before releasing the frontend to users.

@halfwhole @t-cheepeng @AdithyaNarayan please help to fact check what I said above haha

Reply:
>This is a good point. We don't even have a record of what the previous email was. I think the UI need to show the email address in a way it is clear that the tracking is done via the email. We have to be careful because instructors can take action based on the audit log that can affect student marks.

Will raise an issue for this so we can discuss what to tell the instructors on the logs page although I don't think it will be essential. See points below.

> Does any of the above affects the tracking of activities?

Neither cases should be an issue. If they register using google account halfway through the session it changes their google ID but not the email. Similarly, the re-register only affects the google ID but not email. i.e. Student's email remains the same throughout (the one the instructor inputted when enrolling the student)

>please help to fact check what I said above haha
Yup, that is accurate.

Reply:
I see. Yes, we need to inform the user that the tracking is based on the last known email address only and the history earlier than the most recent to the email address will not be shown.